src/bundle/Resources/public/scss/mixins/_inputs.scss

```diff
@@ -0,0 +1,9 @@
+@mixin input-active {
+    border-color: $ibexa-color-primary;
+    box-shadow: 0 0 calculateRem(2px) 0 rgba($ibexa-color-primary, 0.75);
+}
+
+@mixin invalid-text-input {
```

Comment:
I wonder, should we try to keep same order for names? either `input-invalid-text` or `active-input`? As few lines above we start mixin with input, and here input is last word @dew326